ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java

```diff
@@ -25,6 +26,15 @@
     private static final By ADD_NO_KNOWN = By
             .xpath("//*[contains(text(),'No Known Allergy')]");
     private static final By DELETE_NO_KNOWN = By.className("delete-action");
+    public static final By CHOOSEDRUG = By.id("allergen-162299");
+    public static final By CHOOSEREACTION = By.id("reaction-121629");
```

Comment:
are you able to reproduce  these ids (allergen-162299, reaction-121629) both locally and remotely on running openmrs instances?